### ISSUE TITLE
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-FAsGZy/eth-account/

### ISSUE THREAD

**Issue by SabakGaara** [melonproject/oyente#301]
I tried to install oyente using pip2：
```
(env) liyue@oyente ~ $ pip2 install oyente
Requirement already satisfied: oyente in ./env/lib/python2.7/site-packages/oyente-0.2.7-py2.7.egg
Requirement already satisfied: requests in ./env/lib/python2.7/site-packages (from oyente)
Requirement already satisfied: web3 in ./env/lib/python2.7/site-packages/web3-4.0.0-py2.7.egg (from oyente)
Requirement already satisfied: z3-solver in ./env/lib/python2.7/site-packages/z3_solver-4.5.1.0.post2-py2.7-linux-x86_64.egg (from oyente)
Requirement already satisfied: urllib3<1.23,>=1.21.1 in ./env/lib/python2.7/site-packages (from requests->oyente)
Requirement already satisfied: idna<2.7,>=2.5 in ./env/lib/python2.7/site-packages (from requests->oyente)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in ./env/lib/python2.7/site-packages (from requests->oyente)
Requirement already satisfied: certifi>=2017.4.17 in ./env/lib/python2.7/site-packages (from requests->oyente)
Collecting cytoolz<1.0.0,>=0.9.0 (from web3->oyente)
  Using cached cytoolz-0.9.0.1.tar.gz
Collecting eth-abi<2,>=1.0.0 (from web3->oyente)
  Using cached eth-abi-1.0.0.tar.gz
Collecting eth-account<1.0.0,>=0.1.0-alpha.2 (from web3->oyente)
  Using cached eth-account-0.2.0a0.tar.gz
    Complete output from command python setup.py egg_info:
    zip_safe flag not set; analyzing archive contents...
    pypandoc.__init__: module references __file__
    
    Installed /tmp/pip-build-muftol/eth-account/.eggs/pypandoc-1.4-py2.7.egg
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-muftol/eth-account/setup.py", line 70, in <module>
        'Programming Language :: Python :: Implementation :: PyPy',
      File "/home/liyue/env/local/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
        _setup_distribution = dist = klass(attrs)
      File "/home/liyue/env/local/lib/python2.7/site-packages/setuptools/dist.py", line 372, in __init__
        _Distribution.__init__(self, attrs)
      File "/usr/lib/python2.7/distutils/dist.py", line 287, in __init__
        self.finalize_options()
      File "/home/liyue/env/local/lib/python2.7/site-packages/setuptools/dist.py", line 528, in finalize_options
        ep.load()(self, ep.name, value)
      File "/tmp/pip-build-muftol/eth-account/.eggs/setuptools_markdown-0.2-py2.7.egg/setuptools_markdown.py", line 22, in long_description_markdown_filename
        output = pypandoc.convert(markdown_filename, 'rst')
      File "/tmp/pip-build-muftol/eth-account/.eggs/pypandoc-1.4-py2.7.egg/pypandoc/__init__.py", line 66, in convert
        raise RuntimeError("Format missing, but need one (identified source as text as no "
    RuntimeError: Format missing, but need one (identified source as text as no file with that name was found).
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-muftol/eth-account/
```
Specs:

python: 2.7.11+
pip:9.0.3

**Comment by luongnt95** [comment]
Try running: `pip2 install web3==2.7.0` and then `pip2 install oyente`

**Comment by SabakGaara** [comment]
It worked,thanks @luongnt95

**Comment by unboxedtype** [comment]
I had the same problem as mentioned by gulshanvasnani.
So I changed command slightly: 
pip2 install web3==2.7.0 eth-hash==0.1.0
There were several versions of eth-hashed released since 0.1.0; not sure how bad this solution is.

**Comment by dmdque** [comment]
This should be added to the readme.

**Comment by Ooqu4** [comment]
When trying the solution of https://github.com/melonproject/oyente/issues/301#issuecomment-397980803, I now get a further error:

```console
$ pip2 install web3==2.7.0 eth-hash==0.1.0
[...]
Collecting eth-typing<2.0.0,>=1.3.0 (from eth-utils<2,>=1.0.2->rlp>=0.4.6->web3==2.7.0)
  Could not find a version that satisfies the requirement eth-typing<2.0.0,>=1.3.0 (from eth-utils<2,>=1.0.2->rlp>=0.4.6->web3==2.7.0) (from versions: )
No matching distribution found for eth-typing<2.0.0,>=1.3.0 (from eth-utils<2,>=1.0.2->rlp>=0.4.6->web3==2.7.0)
```

#### Solution:
Additionally specify the version of `eth-utils` to be `1.0.1`.

```console
 pip2 install web3==2.7.0 eth-hash==0.1.0 eth-utils==1.0.1 oyente
```

Note: I chose version number `1.0.1` because it was released roughly at the same time as version `0.1.0` of `eth-hash`, and I guessed that those versions are probably compatible.
